_playwright-tests/helpers/loginHelpers.ts

```diff
@@ -100,6 +114,10 @@ export const throwIfMissingEnvVariables = () => {
     'BASE_URL',
     'ADMIN_USERNAME',
     'ADMIN_PASSWORD',
+    'LAYERED_REPO_ACCESS_USERNAME',
+    'LAYERED_REPO_ACCESS_PASSWORD',
+    'RHEL_ONLY_ACCESS_USERNAME',
+    'RHEL_ONLY_ACCESS_PASSWORD',
```

Comment:
these should also be needed only when the integration flag is set, similar to [this](https://github.com/content-services/content-sources-frontend/pull/601/files#r2256295864), you can just move them few lines down 🚚😅